src/command_modules/azure-cli-eventgrid/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+0.2.1
++++++
+* Added support for specifying deadletter destination for an event subscription.
+* Added support for using storage queue as a destination for an event subscription.
+* Added support for using hybrid connection as a destination for an event subscription.
+* Added support for specify retry policy parameters for an event subscription.
+* Added a warning message for manual handshake validation when Webhook as destination is used for an event subscription.
+* Added source-resource-id parameter for all event subscription related commands and mark all other source resource related parameters as deprecated.
+* Enhance the event subscription list commands to resolve usability issues.
```

Comment:
Typically this would look a little more command specific. Examples:
```
* `event-subscription create/update`: Added `--deadletter-endpoint`.
* `event-subscription create/update`: Added `--source-resource`. Deprecated `--blah`, `--bloo`.
```

Reply:
@tjprescott @williexu .. It seems we are hitting CLI build issue that does not seem to be related to our change or module. Can you please help us resolving this one?